Assignment2Conditionals/src/assignment2conditionals/Assignment2Conditionals.java

```diff
@@ -0,0 +1,107 @@
+package assignment2conditionals;
+
+import java.util.Scanner;
+
+public class Assignment2Conditionals 
+{
+
+    public static void main(String[] args) 
+    {
+     Scanner scan = new Scanner (System.in);
+           
+     System.out.println("\t*************************************\n"
+                      + "\t*       Steve's Calculator          *\n"
+                      + "\t*************************************");
+          
+     System.out.println(" 1: Math\n 2: Subtraction\n 3: Multiplication"
+                     + "\n 4: Devision\n 5: Word Counter\n 6: Exit Program\n");
+     
+  
+     System.out.print("Please Pick An Option: ");
+     int userInputNum = scan.nextInt();
+     System.out.print("\n");
+      
+     
+     if(userInputNum == 1)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 + var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+      
+     else if(userInputNum == 2)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 - var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+     
+     else if(userInputNum == 3)
+      {
+        System.out.print("Please enter in your First variable: ");
+        double var1 = scan.nextDouble();
+        System.out.print("Please enter in your Second variable: ");
+        double var2 = scan.nextDouble();
+        double product = var1 * var2;
+        System.out.println("The answer is: " + product + "\n");
+      }
+     
+     else if(userInputNum == 4)
```

Comment:
Continuing with use of indentation style https://github.com/WCCCEDU/twitter-commons/blob/master/src/java/com/twitter/common/styleguide.md#indent-style

we should use the **cuddle** style of _if else_ sequences

```Java
if(condition){
  amazing();
} else if(otherCondition){
  noop();
} else {
  noop();
}

Cuddle means the braces cuddle the else portions